packages/api/src/balance/balance.entity.ts

```diff
@@ -24,9 +25,9 @@ export class Balance extends BaseEntity {
   public readonly balance: string;
 
   @AfterLoad()
-  populateEthToken() {
-    if (this.tokenAddress === ETH_TOKEN.l2Address && !this.token) {
-      this.token = ETH_TOKEN;
+  async populateBaseToken() {
+    if (!this.token && this.tokenAddress.toLowerCase() === baseTokenData.l2Address) {
```

Comment:
Previously, `ETH_TOKEN.l2Address` was hard coded and we knew it was in lower case. With `baseTokenData.l2Address` being configurable, we need to `toLowerCase()` it too.